contracts/JetStakingV2.sol

```diff
@@ -815,6 +896,7 @@ contract JetStakingV1 is AdminControlled {
     function getRewardsAmount(uint256 streamId, uint256 lastUpdate)
```

Comment:
we can store `block.timestamp` in a variable, as it is used many times. This will save gas consumption

Reply:
Not sure if we can do this in the current upgrade (having lots of changes might delay the upgrade).

Reply:
updating it.